.github/workflows/deploy-frontend.yml

```
@@ -27,14 +27,22 @@ jobs:
 
       - name: Install dependencies
-        working-directory: ./frontend
+        working-directory: ./ch11-frontend # Corrected path
         run: npm ci
 
+      - name: Lint frontend code
+        working-directory: ./ch11-frontend # Corrected path
+        run: npm run lint
+
+      - name: Run frontend tests
+        working-directory: ./ch11-frontend # Corrected path
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)